package.json

```diff
@@ -13,9 +13,12 @@
   },
   "dependencies": {
     "@auth0/nextjs-auth0": "^1.8.0",
+    "@next/font": "13.1.6",
+    "@swc/helpers": "^0.4.14",
```

Comment:
Unfortunately we won't be able to use `@next/font` since our Next.js version is too low. I think we can remove `@swc/helpers` as well.

```suggestion

```